Quantum-Control-Applications-QuAM/Superconducting/configuration/make_quam.py

```diff
@@ -46,17 +48,17 @@ def create_quam_superconducting(
         machine.wiring = wiring
     else:
         raise ValueError("Wiring must be provided.")
-
```

Comment:
Agreed, everything that is installation-specific should be moved out of the function.